api_tests/test_process_table.py

```diff
@@ -0,0 +1,25 @@
+"""
+Tests of attempts to access the process table.
+
+These should all be denied. Accessing the process table could reveal the
+directory in use for another active code execution, and we don't have any
+protection against that other than unpredictable directory names.
```

Comment:
Too much info?

Reply:
I don't think so. It's needed for explaining why these tests exist in the first place. We also have several protections against exploit.